### ISSUE TITLE
(One#26775002) [Configuration window - Advanced Parameters]: Ensures every ARIA input field has an accessible name.

### ISSUE THREAD

**Issue by ek68794998** [Azure/LogicAppsUX#4297]
### **User Experience:** 

If these names are missing or inadequate, screen reader users may hear generic or uninformative labels, hindering their ability to make informed decisions and engage with your content. Without them, users may not know what a button, link, or menu item does, making it difficult to interact with and navigate your application effectively.

### **Repro Steps:** 

1. Open URL [https://make.powerautomate.com/](<https://make.powerautomate.com/>) and sign in with Valid credentials. 
2. Flow Homepage will appear on the screen, navigate to 'My Flows' present on left nav and select it. 
3. Find a flow and click on 'Edit' 
4. Edit page will appear, navigate to 'New Step' button and select it. 
5. Choose an Action section will appear on the screen, navigate to any Trigger e.g.: When a new email arrives present in section and select 'Trigger' from the Action Type dropdown and select the trigger. 
6. Configuration screen will appear with controls like 'Folder' show advanced options etc. 
7. Open Accessibility Insights for Web and run Fast Pass. 
8. Verify whether every ARIA input field has an accessible name or not. 

### **Actual Result:** 

Aria-label and title attribute are not defined for combo box controls available under Parameters tab

### **Expected Result:** 

Aria-label and title attribute should be defined for combo box controls available under Parameters tab.

### **Issue:** 

Ensures every ARIA input field has an accessible name (aria-input-field-name - [https://accessibilityinsights.io/info-examples/web/aria-input-field-name](<https://accessibilityinsights.io/info-examples/web/aria-input-field-name>) )

### **Target application:** 

Manage your flows \| Power Automate - [https://make.preview.powerautomate.com/environments/839eace6-59ab-4243-97ec-a5b8fcc104e4/flows/3b6866d6-01d0-4505-a5c3-7d51330eaf7b?v3=true](<https://make.preview.powerautomate.com/environments/839eace6-59ab-4243-97ec-a5b8fcc104e4/flows/3b6866d6-01d0-4505-a5c3-7d51330eaf7b?v3=true>)

### **Element path:** 

\#Dropdown5926

### **Snippet:** 

```
<div data-is-focusable="true" data-ktp-target="true" id="Dropdown5926" tabindex="0" role="combobox" aria-haspopup="listbox" aria-expanded="false" class="ms-Dropdown dropdown-1173">
```

### **How to fix:** 

**Fix any of the following:**

- aria-label attribute does not exist or is empty 
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty 
- Element has no title attribute 

Environment: Microsoft Edge version 121.0.0.0

\====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](<http://aka.ms/AccessibilityInsights>).

### **MAS Reference:** 

[MAS 4.1.2 -- Name, Role, Value](<https://aka.ms/MAS4.1.2>)

### **ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:** 

N/A

### **DOM details available:** 

Yes

### **Reference Links:** 

- [Accessibility Insights!](<https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0>) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

![MAS4 1 2 Ensures every ARIA input field has an accessible name](<https://github.com/Azure/LogicAppsUX/assets/1350074/401d77d9-571f-4d4f-a64d-514b8f21ca9f>)

### Browser 

- **OS Version:** Windows 11 Version 23H2 (OS Build 22631.3007) 
- **Browser:** Edge Version 120.0.2210.144 (Official build) (64-bit) 
- **\#URL:**[https://make.powerautomate.com/](<https://make.powerautomate.com/>) 
- **Screen Reader: **Narrator 

### Additional context 

[One#26775002](<https://dev.azure.com/msazure/One/_workitems/edit/26775002>)

**Comment by hartra344** [comment]
This was already done: 

https://github.com/Azure/LogicAppsUX/assets/13208452/ce918f2a-da7c-4dd9-8d6c-94272fbc04a5


PR: #4090
